.storybook/main.js

```diff
@@ -16,8 +16,7 @@ module.exports = {
       ...config,
       module: {
         ...config.module,
-        // Replace default storybook rules with our webpack rules.
-        rules: webpackConfig.module.rules,
+        rules: [...config.module.rules, ...webpackConfig.module.rules],
```

Comment:
replacing instead of spreading these rules broke the [Docs addon](https://storybook.js.org/addons/@storybook/addon-docs) (which comes default with @storybook/addon-essentials)